CS410Assignment2/src/sensor/SensorAdapter.java

```diff
@@ -0,0 +1,14 @@
+package sensor;
+
+import java.awt.Color;
+
+public interface SensorAdapter {
+	
+	public String getReport();
+	public Double getValue();
+	public String getType();
+	public Color getColor();
+	public int getMax();
```

Comment:
using getMax here for drawing is useful for drawing the progress bars. I hadn't considered that, I just drew rectangles and had a scaling factor. Jprogressbar seems like an overall much more efficient way to do this.